test/kubemark/start-kubemark.sh

```diff
@@ -47,22 +47,115 @@ gcloud compute instances create ${MASTER_NAME} \
     --scopes "storage-ro,compute-rw,logging-write" \
     --disk "name=${MASTER_NAME}-pd,device-name=master-pd,mode=rw,boot=no,auto-delete=no"
 
-MASTER_IP=`gcloud compute instances describe hollow-cluster-master --zone=${ZONE} | grep networkIP | cut -f2 -d":" | sed "s/ //g"`
+MASTER_IP=$(gcloud compute instances describe hollow-cluster-master --zone=${ZONE}| grep natIP: | cut -f2 -d":" | sed "s/ //g")
+
+ensure-temp-dir
+gen-kube-bearertoken
+create-certs ${MASTER_IP}
+
+CA_CERT=${CA_CERT_BASE64}
```

Comment:
Why are you re-assigning all of these variables? Giving them shorter names doesn't seem to add much value, given that below you run them through `base64 -d` where it might actually be nice to see that you are decoding a variable that is currently base64 encoded.